Header header

int8 power
float32 rotation
float32 x_translation
float32 y_translation
int8 green_led
int8 red_led
